include/glow/Importer/ONNXIFIModelLoader.h

```diff
@@ -56,20 +59,14 @@ class ONNXIFIModelLoader : public ONNXModelLoader {
 
   /// \returns a unique_ptr<ONNXIFIModelLoader> if \p onnxModel can be
   /// parsed and static weights can be loaded from the \p wightDescriptors.
-  /// \returns Error otherwise.
+  /// \returns Error otherwise. \p loadInputsAsPlaceholders is passed to
+  /// loadInputs to determine whether inputs are loaded as Placeholders or
+  /// Tensors. Loading as Tensors is useful for when the graph being loaded is
+  /// actually a small slice of a larger graph.
   static llvm::Expected<std::unique_ptr<ONNXIFIModelLoader>>
```

Comment:
btw, can we get rid of unique_ptr for ONNXIFIModelLoader? (the only use case was that we'd check whether it's null or not, but now we have Expected wrapper)

Reply:
We can't because returning a `llvm::Expected<T>` still causes a copy of the underlying `T`, which in most cases is fine and cheap because the expected value is a pointer, int, float, etc but here it would cause issues because it would be expensive and `ONNXIFIModelLoader` owns tensors